scaleway/data_source_instance_security_group.go

```diff
@@ -0,0 +1,76 @@
+package scaleway
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+)
+
+func dataSourceScalewayInstanceSecurityGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceScalewayInstanceSecurityGroupRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Should we read rules ?

Reply:
I saw current security group data source has the 2 following computed: 
```go
			"description": {
				Type:     schema.TypeString,
				Computed: true,
			},
			"enable_default_security": {
				Type:     schema.TypeBool,
				Computed: true,
			},
```
My question is: how much data do we want compute on a data source? 🤔